Deactivate ufb, this needs the UAPI Linux kernel headers from >= 3.18, 
this tools is currently not used, so just remove it for now.

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 CC ?= $(CROSS_COMPILE)gcc
 BINDIR ?= /usr/sbin
-PROGRAMS = uuc sdimage ufb
+PROGRAMS = uuc sdimage
 LIBS ?= -lpthread
 
 all: $(PROGRAMS)
